r/R/table.R

```diff
@@ -175,6 +175,16 @@ Table$create <- function(..., schema = NULL) {
     return(dplyr::group_by(out, !!!dplyr::groups(dots[[1]])))
   }
   
+  # If any arrays are length 1, recycle them  
+  arr_lens <- map(dots, length)
+  if (length(dots) > 1 && any(arr_lens == 1) && !all(arr_lens==1)){
+    dots <- purrr::modify2(
```

Comment:
In `arrow-package.R`, add `modify2` to the line beginning with `#' @importFrom purrr`. Then do `devtools::document()` to update the `NAMESPACE` file. Then remove the `purrr::` from this line.